samcli/commands/test_runner/init/cli.py

```diff
@@ -14,19 +14,22 @@
 from samcli.lib.utils.boto_utils import BotoProviderType
 from samcli.lib.utils.colors import Colored
 
-SHORT_HELP = "Generates a Test Runner CloudFormation Template in YAML format, as well as a resource-ARN map."
+SHORT_HELP = """
+Generates a Test Runner CloudFormation Template file in YAML format. It also generates a resource-ARN map YAML file, where the keys are your resource
+names in the format of an environment variable name, and the values are the corresponding resource ARN."""
+
 HELP_TEXT = """
 This command generates a CloudFormation Template file that can be deployed to instantiate a Test Runner Stack.
 
-It also generates a resource-ARN map, where the keys are your resource names in the format of an environment variable name, and the values
+It also generates a resource-ARN map YAML file, where the keys are your resource names in the format of an environment variable name, and the values
 are the corresponding resource ARN. 
 
 For example, 
 
 _MY_LAMBDA_FUNCTION : arn:aws:lambda:us-east-1:123456789012:function:my-lambda-function
 
-This resource-ARN map can be passed to the `sam test-runner run` command which will expose these mappings as environment variables to the Test Runner Fargate instance,
-and thus can be used in test code.
+This file can be passed to `sam test-runner run` (using the `--container-vars` option), to expose the mappings as environment variables to the Fargate container.
+These environment variables can be picked up and used by your test code.
```

Comment:
> can be picked up and

Is this wording precise enough? It might be better to phrase like "These environment variables are available to your test suite" or "These environment variables are exported to your test suite."

Reply:
Yeah, I agree it's a bit too colloquial. I like the "These environment variables are available to your test suite", but we should probably stay consistent with the `exposed` verb, so maybe "These environment variables are exposed to your testsuite"?

Reply:
@bentvelj Either of them sound fine honestly. I would pick the one you think it best. No need to overthink it, it's something we can always adjust later too. :)

Reply:
Yep! Easy change, fixed to "These environment variables are exposed to your testsuite" :)